js/msp/MSPHelper.js

```diff
@@ -463,22 +463,33 @@ var mspHelper = (function (gui) {
                 console.log("Servo mix saved");
                 break;
             case MSPCodes.MSP2_INAV_LOGIC_CONDITIONS:
-                LOGIC_CONDITIONS.flush();
```

Comment:
@Scavanger this is a great PR. But we have to change how it's done. I've left a review in the INAV repo but it ultimately goes down to:

1. Leave `MSP2_INAV_LOGIC_CONDITIONS` unchanged
1. Add `MSP2_INAV_LOGIC_CONDITIONS_SINGLE` frame
1. Switch configurator to use `MSP2_INAV_LOGIC_CONDITIONS_SINGLE`

And then, in one of future releases we can remove support for `MSP2_INAV_LOGIC_CONDITIONS` completely.

What do you think?